### DISCUSSION TITLE
Sx1262 frequency setting?

### DISCUSSION THREAD

**Discussion by duzycinek** [jgromes/RadioLib#224] [Q&A] [answered]
Hello, I got question. How to set up frequency of module to exact between 869,4 and 869,85 MHz by library? Thanks!

**Comment by jgromes** [comment, marked as answer]
Well - which one then? 869.4 or 896.85? SX126x can set frequency in 1 Hz steps, so you can do `setFrequency(896.123456)` and it will set it's frequency to 896.123456 MHz (within the XTAL/TCXO error, of course).

**Reply by duzycinek** [reply]
Alright, that is what i needed. Thank you so much. :)
